.changeset/tender-cherries-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/icons': major
```

Comment:
Hmm, this should be a `minor`. Can you just tweak this real quick please?

Reply:
Fixed that @TheSisb - didn't realise that the CLI accepted an empty selection for the prompt

Reply:
Yeah it's a little confusing. Thanks!